modules/ppcp-settings/resources/js/hooks/usePaymentGatewaySync.js

```diff
@@ -19,8 +19,8 @@ export const usePaymentGatewaySync = () => {
 	const { isReady: merchantIsReady } = CommonHooks.useStore();
 
 	const [ isSyncing, setIsSyncing ] = useState( false );
-	const [ syncCompleted, setSyncCompleted ] = useState( false );
-	const [ syncError, setSyncError ] = useState( null );
+	const [ , setSyncCompleted ] = useState( false );
+	const [ , setSyncError ] = useState( null );
```

Comment:
Hm... looks like we do not use those two states. There's no point in setting the state but not using the value anywhere.

To clean this up, the two states should be completely removed from the code.

Possibly, those states are a left over from a previous change